velox/exec/HashProbe.cpp

```diff
@@ -178,6 +182,43 @@ void HashProbe::initializeFilter(
   filterInputType_ = ROW(std::move(names), std::move(types));
 }
 
+void HashProbe::prepareForNullAwareAntiJoinWithFilter() {
+  folly::F14FastMap<column_index_t, column_index_t> filterColumns(
+      filterBuildInputs_.size());
+  for (auto& p : filterBuildInputs_) {
+    filterColumns[p.inputChannel] = p.outputChannel;
+  }
+  auto rows = table_->rows();
+  RowContainerIterator iter;
+  std::vector<char*> data(rows->numRows());
+  auto numRows = rows->listRows(
```

Comment:
Are we reading all the rows from the build side? This can be a lot of rows and they may not fit in a single RowVector.